docs/ref/settings.txt

```diff
@@ -3208,7 +3216,14 @@ Possible values for the setting are:
   regular link from an external website and be blocked in CSRF-prone request
   methods (e.g. ``POST``).
 
-* ``None``: disables the flag.
+* ``'None'``: the session cookie will be sent with all same-site and cross-site
+  requests.
+
+* ``False``: disables the flag.
```

Comment:
The ``SameSite`` flag can still be disabled by using `None`, but I I thought that ``False`` is less confusing with ``'None'``.

Reply:
Oh my github didn't autoupdate :/ Yes `False` is certainly better. I wonder if we want to raise a warning if it is set to None. I'll have to look up the difference on what happens if same_site is not set vs it is set to `'None'`. Ie what are browsers going to default same_site to if it is not explicitly set?

Reply:
Okay, according to https://caniuse.com/#search=samesite
> 3 Implemented Incrementally Better Cookies by default: Cookies without SameSite are treated as Lax, SameSite=None cookies without Secure are rejected.
This means if someone sets it to `None` (sic) by accident instead of `'None'` they will get a stricter behaviour than what specified -- this is imo okay security wise (usability wise it might be annoying, but security wins ;))

Reply:
They should default to `'None'`:

> If the cookie-attribute-list contains an attribute with an attribute-name of "SameSite", set the cookie's same-site-flag to attribute-value (i.e. either "Strict", "Lax", or "None"). Otherwise, set the cookie's same-site-flag to "None".

> If the "SameSite" attribute's value is something other than these three known keywords, the attribute's value will be treated as "None".

but e.g. future releases of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`.

Reply:
OK, so `Lax` is a default value in Chrome 80+, in [Firefox 69+](https://bugzilla.mozilla.org/show_bug.cgi?id=1551798), and probably a default value will change also in a future release of Edge (maybe 80+).